src/components/atoms/tooltip/index.tsx

```diff
@@ -32,8 +32,8 @@ const Tooltip = ({
         onOpenChange={onOpenChange}
         delayDuration={delayDuration}
       >
-        <RadixTooltip.Trigger onClick={onClick} type="button">
-          {children}
+        <RadixTooltip.Trigger onClick={onClick} asChild={true}>
+          <span>{children}</span>
         </RadixTooltip.Trigger>
         <RadixTooltip.Content
           side={side ?? "bottom"}
```

Comment:
Is there a reason why we want to keep the Tooltip as a button? The reason I am asking is that it's causing quite a few React warnings across the project because we in some cases have tooltips nested within buttons, and since buttons can't be nested within buttons we get several errors. in the console output 😄